nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ScanAttribute.java

```diff
@@ -99,12 +114,26 @@
             .defaultValue(null)
             .build();
 
+    private static final Validator characterValidator = new StandardValidators.StringLengthValidator(1, 1);
+
+    public static final PropertyDescriptor DICTIONARY_ENTRY_METADATA_DEMARCATOR = new PropertyDescriptor.Builder()
+            .name("dictionary-entry-metadata-demarcator")
+            .displayName("Dictionary Entry Metadata Demarcator")
+            .description("A single character used to demarcate the dictionary entry string between dictionary value and metadata.")
```

Comment:
Considering adding to the description what would happen if no demarcator was provided.  For example, "If no demarcator is specified then there will be no hit metadata and the entire dictionary term will be matched."